tests/Integration/all_integration_server_tests.sh

```diff
@@ -85,4 +81,5 @@ echo -e "*** $(date -u)  **** Resources TESTS ****\n"
 python $SERVERINSTALLDIR/DIRAC/tests/Integration/Resources/Storage/Test_Resources_GFAL2StorageBase.py ProductionSandboxSE 2>&1 | tee -a $SERVER_TEST_OUTPUT; (( ERR |= $? ))
 
 # Can only run if there's a Stomp MQ local...
+# TODO Enable
 # python -m pytest $SERVERINSTALLDIR/DIRAC/tests/Integration/Resources/MessageQueue/Test_ActiveClose.py 2>&1 | tee -a $SERVER_TEST_OUTPUT; (( ERR |= $? ))
```

Comment:
Should I open an issue to add another service to the docker-compose file so this test can be ran? Or is there a reason to not run it?

Reply:
No, as right now I am not even sure this test can run at all.